server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -67,38 +75,43 @@
 
     private final CommandStatusService commandStatusService;
 
-    @Nullable
     private final CommandScheduler scheduler;
 
+    private final ProblemLog problemLog;
+
     /**
      * Is {@code true} if the Bounded Context (to which this Command Bus belongs) is multi-tenant.
      * Therefore commands posted to this Command Bus must have the {@code namespace} attribute.
      */
     private boolean isMultitenant;
 
-    private ProblemLog problemLog = new ProblemLog();
-
-    private CommandBus(Builder builder) {
-        commandStore = builder.getCommandStore();
-        scheduler = builder.getScheduler();
-        commandStatusService = new CommandStatusService(commandStore);
-        rescheduleCommands();
-    }
-
-    private void rescheduleCommands() {
-        final Iterator<Command> commands = commandStore.load(SCHEDULED);
-        // TODO:2016-05-12:alexander.litus: re-calculate delays and schedule
-        /*while (commands.hasNext()) {
-            final Command command = commands.next();
-            schedule(command);
-        }*/
+    /**
+     * Creates a new instance.
+     *
+     * @param commandStore a store to save commands
+     * @return a new instance
+     */
+    public static CommandBus newInstance(CommandStore commandStore) {
+        final CommandScheduler scheduler = createCommandScheduler();
+        final ProblemLog log = new ProblemLog();
+        final CommandBus commandBus = new CommandBus(checkNotNull(commandStore), scheduler, log);
+        return commandBus;
     }
 
     /**
-     * Creates a new builder for command bus.
+     * Creates a new instance.
+     *
+     * @param commandStore a store to save commands
+     * @param scheduler a command scheduler
+     * @param problemLog a problem logger
      */
-    public static Builder newBuilder() {
```

Comment:
Please revert to having builder. Have also setting to multitenancy attribute in the builder and remove the setter. There is no need to change the attribute after the instance is constructed.